eclipse-theia/theia
# All three brackets for variables

fixed #14464

#### What it does

Allows to use three brackets for variables in prompts. No mixed formats are allows for consistency

#### How to test

There are automated test.
To manually test, 
- open the code completion prompt and a code file. 
- Add one bracket to {{textUntilCurrentPosition}}, see how it fails. 
- Add the second bracket to complete {{{textUntilCurrentPosition}}} and see how it works again. 
- Check that the variable is still listed in the AI configuration view then.
- Check that in the prompt editor, three brackets are valid, but mixtures are an error

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
